docs/api/main-config/main-config-build.mdx

```diff
@@ -28,10 +28,10 @@ Type: `TestBuildFlags`
 }
 ```
 
-Configures Storybook's production builds for performance testing purposes by disabling certain features from the build. When running `build-storybook`, this feature is enabled by setting the `--test` [flag](../cli-options.mdx#build).
+Configures Storybook's production builds for performance testing purposes by disabling certain features from the build. When running `storybook build`, this feature is enabled by setting the `--test` [flag](../cli-options.mdx#build).
 
 <Callout variant="info" icon="💡">
-  The options documented on this page are automatically enabled when the `--test` flag is provided to the `build-storybook` command. We encourage you to override these options only if you need to disable a specific feature for your project or if you are debugging a build issue.
+  The options documented on this page are automatically enabled when the `--test` flag is provided to the `storybook build` command. We encourage you to override these options only if you need to disable a specific feature for your project or if you are debugging a build issue.
 </Callout>
```

Comment:
@ahayes91, when you have a moment, can you adjust this to:
```md

<Callout variant="info" icon="💡">

  The options documented on this page are automatically enabled when the `--test` flag is provided to the [`storybook build`](../cli-options.mdx#build) command. We encourage you to override these options only if you need to disable a specific feature for your project or if you are debugging a build issue.

</Callout>
```

The rationale here is to provide some additional context, via a link to this option and the other options that the user has at his disposal when issuing the command.